namesrv/src/main/java/org/apache/rocketmq/namesrv/processor/DefaultRequestProcessor.java

```diff
@@ -240,6 +240,13 @@ public RemotingCommand registerBrokerWithFilterServer(ChannelHandlerContext ctx,
             registerBrokerBody.getFilterServerList(),
             ctx.channel());
 
+        if (result == null) {
```

Comment:
The `result` seems can not null, it is initialized in first line, please correct me if I understand error

> public RegisterBrokerResult registerBroker(
            final String clusterName,
            final String brokerAddr,
            final String brokerName,
            final long brokerId,
            final String haServerAddr,
            final TopicConfigSerializeWrapper topicConfigWrapper,
            final List<String> filterServerList,
            final Channel channel) {
        RegisterBrokerResult result = new RegisterBrokerResult();
        try {
            try {
                this.lock.writeLock().lockInterruptibly();

Reply:
```java
if (topicConfigWrapper.getTopicConfigTable().size() == 1) {
    log.warn("Can't register topicConfigWrapper={} because broker={} has not registered.", topicConfigWrapper, brokerName);
    return null; 
}
```

`null` is returned directly. @Oliverwqcwrw

Reply:
ok, I didn't notice you added this piece of code, thanks for your reply